lib/stat_tracker.rb

```diff
@@ -22,13 +25,13 @@ def self.from_csv(locations)
 
   def highest_total_score
     games.max_by do |game|
-      game.away_goals + game.home_goals
+      self.calculate_total_score(game)
```

Comment:
Harrison - we may want to change this back to the original because i think we may need to make the iterations into modules. Cool shortcut tho.